src/components/MDXComponents/MDXCopyCodeButton.tsx

```diff
@@ -0,0 +1,61 @@
+import { useState } from 'react';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import { Button, VisuallyHidden } from '@aws-amplify/ui-react';
+import { trackCopyClicks } from '@/utils/track';
+
+export const prepareCopyText = (codeString: string): string => {
```

Comment:
I had trouble figuring out a way to test the copied text from just inside the button component (which is wrapped in CopyToClipboard) so I pulled it out into a utility to test the regex replace by itself.